efficientdet/keras/efficientdet_keras.py

```diff
@@ -26,7 +26,7 @@
 from keras import fpn_configs
 from keras import postprocess
 from keras import util_keras
-from keras import model_optimization
+from keras.model_optimization import get_method as tfmot_method
```

Comment:
the standard format is "from folders import module_name".
My suggestion is to rename model_optimization.py as "tfmot.py", then here you can do "from keras import tfopt", and later you can call: tfopt.get_method(method)

Reply:
I try to ignore import errors when I import the module on the environment which didn't install tensorflow_model_optimization.
Any better solution?